lib/ex_unit/lib/ex_unit/case.ex

```diff
@@ -141,20 +141,30 @@ defmodule ExUnit.Case do
   therefore reserved:
 
     * `:module`     - the module on which the test was defined
+
     * `:file`       - the file on which the test was defined
+
     * `:line`       - the line on which the test was defined
+
     * `:test`       - the test name
+
     * `:async`      - if the test case is in async mode
+
     * `:registered` - used for `ExUnit.Case.register_attribute/3` values
+
     * `:describe`   - the describe block the test belongs to
 
   The following tags customize how tests behave:
 
     * `:capture_log` - see the "Log Capture" section below
+
     * `:skip` - skips the test with the given reason
+
     * `:timeout` - customizes the test timeout in milliseconds (defaults to 60000).
       Accepts `:infinity` as a timeout value.
 
+    * `:tmp_dir` - (since v1.11.0) see the "Tmp Dir" section below
```

Comment:
I would advice not mention the `@since` for attributes, as more features are added to the language, it will become noise in the docs.